core-rust/core-api-server/core-api-schema.yaml

```diff
@@ -7072,6 +7144,184 @@ components:
                 type: string
               message:
                 type: string
+############################
+# REQUEST: /stream/proofs #
+###########################
+    StreamProofsRequest:
+      description: |
+        A request to retrieve a sublist of proofs.
+      type: object
+      required:
+        - network
+        - filter
+      properties:
+        network:
+          $ref: "#/components/schemas/NetworkIdentifier"
+        filter:
+          $ref: "#/components/schemas/StreamProofsFilter"
+        max_page_size:
+          description: If specified, the maximum number of proofs that will be returned.
+          type: integer
+        continuation_token:
+          $ref: "#/components/schemas/ContinuationToken"
+    StreamProofsFilterType:
+      type: string
+      enum:
+        - All
+        - NewEpochs
+        - ProtocolUpdateTriggers
+        - ProtocolUpdateExecution
+    StreamProofsFilter:
+      type: object
+      required:
+        - type
+      properties:
+        type:
+          $ref: "#/components/schemas/StreamProofsFilterType"
+      discriminator:
+        propertyName: type
+        mapping:
+          # NOTE: These need to match StreamProofsFilterType
+          All: '#/components/schemas/StreamProofsFilterAll'
+          NewEpochs: '#/components/schemas/StreamProofsFilterNewEpochs'
+          ProtocolUpdateInitializations: '#/components/schemas/StreamProofsFilterProtocolUpdateInitializations'
+          ProtocolUpdateExecution: '#/components/schemas/StreamProofsFilterProtocolUpdateExecution'
+    StreamProofsFilterAll:
+      allOf:
+        - $ref: "#/components/schemas/StreamProofsFilter"
+        - type: object
+          properties:
+            from_state_version:
+              type: integer
+              format: int64
+              minimum: 0
+              maximum: 10000000000
```

Comment:
I believe `state_version` maxes out at `100000000000000` (applies to the `description` too)

Reply:
Fixed 👍